package.json

```diff
@@ -33,7 +33,9 @@
   },
   "devDependencies": {
     "@jest/globals": "29.7.0",
+    "husky": "^8.0.0",
     "jest": "29.7.0",
+    "lint-staged": "^15.2.9",
```

Comment:
Noting that at the time being including these in devDependencies ends up baking them into our final pepr artifact/what runs in cluster, see https://github.com/defenseunicorns/pepr/issues/1103

Reply:
@mjnagel the linked issue doesn't seem to be going anywhere